src/components/Pagination.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Button from './Button';
+
+const PaginationItem = ({ pageNumber, isActive, onChange }) => {
+  const onClick = (event) => {
+    event.preventDefault();
+    onChange(pageNumber);
+  };
+  return (
+    <li className='page-item mr-1'>
+      <Button type={isActive ? 'outline' : 'primary'} onClick={onClick}>
+        {pageNumber + 1}
+      </Button>
+    </li>
+  );
+};
+
+const Pagination = ({ currentPageNumber, totalNumberOfPages, onChange }) => {
+  if (totalNumberOfPages <= 1) {
+    return null;
+  }
+
+  const pageItems = Array.from({
+    length: totalNumberOfPages,
+  }).map((_, pageNumber) => (
+    <PaginationItem
+      key={pageNumber}
+      pageNumber={pageNumber}
+      isActive={currentPageNumber === pageNumber}
```

Comment:
As mentioned above, the parent should know which child is the active one, so the logic lives here